studiopress/genesis-sample
# Feature/Starter Packs

**Summary of change:**

- Addition of Starter Pack content examples. For these starter packs, the options are black & white or color images for the homepage content. The color pack also replaces `#333` with `#111` for the homepage sections.
- Removes the blocks.php template in favor of per-page options.

**This PR has been:**

**Have the changes in this PR been added to the documentation for this project?**
This will require updated theme documentation. cc @susannelson 

**How to test:**

1. Install Genesis 3.1 (develop branch) or Genesis 3.1 beta (once released).
2. Install the `feature/homepage-meta` Genesis Sample branch.
3. Install the Black & White Starter Pack.
4. Verify the homepage content is imported and editable.
5. Verify the homepage content does not show the title or breadcrumbs and is full-width.
6. Verify the menu imported correctly.
7. Verify other content imported correctly by clicking on menu links.
8. Verify the footer widgets imported correctly.
9. Repeat with the Color pack.

**See:** https://github.com/studiopress/genesis-sample/issues/275

**Suggested Changelog Entry:**
Added: Option to import Starter Packs containing either color or black & white images.

_Note:_ This branch started with updating the homepage meta only, but it made sense to do that as part of starter packs.

We may wish to replace the image on the About page with a color one in the color pack. If so, we'll need to update the pack descriptions.
